src/lib/iptables.ts

```diff
@@ -291,3 +303,31 @@ async function applyRules(rules: Rule | Rule[], adaptor: RuleAdaptor) {
 
 	await adaptor(processedRules);
 }
+
+class StdError extends TypedError {}
+
+export async function getRulePosition(
+	chain: string,
+	ruleMatch: string,
+): Promise<number> {
+	const cmd = `iptables -L ${chain} --line-numbers`;
```

Comment:
Are the results of this command consistent across OS versions? This goes with the above comments.

Reply:
`--line-numbers` is supported all the way back to 1.1.2, the oldest version with source code I could find from [this project archive](https://netfilter.org/projects/iptables/downloads.html). 1.1.2 was released in year 2000 so I think we're good, especially because `iptables` itself adds the line numbers to the results of the `-L ${chain}` query.

Reply:
The iptables output is stable indeed as it is regularly used for parsing.